### ISSUE TITLE
Version 2.0.2 not available on npmjs.org

### ISSUE THREAD

**Issue by hypery2k** [webpack-contrib/karma-webpack#213]
**I'm submitting a bug report**

On [npmjs.org](https://www.npmjs.com/package/karma-webpack) version 2.0.2 is not available.

**Webpack version:**
2.2.0

**Webpack Karma version**:
2.0.2

**Karma version**:
1.3.0

**Please tell us about your environment:**
OSX 10.12
**Current behavior:**

```
npm ERR! notarget No compatible version found: karma-webpack@2.0.2
npm ERR! notarget Valid install targets:
npm ERR! notarget 2.0.1, 2.0.0, 1.8.1, 1.8.0, 1.7.0, 1.6.0, 1.5.1, 1.5.0, 1.4.0, 1.3.1, 1.3.0, 1.2.2, 1.2.1, 1.1.0, 1.0.2, 1.0.1, 1.0.0
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
```

**Expected/desired behavior:**

Should install version 2.0.2

**Comment by d3viant0ne** [comment]
@hypery2k - We are aware. When 2.0.1 was published, it was set to a range which has to be fixed before 2.0.2 can be published and the list of those with npm access to publish is short.

I'm going to leave this open for others for the time being, should be able to get this taken care of by this evening.

**Comment by d3viant0ne** [comment]
Version `2.0.2` has been published to npm
